content/src/fieldConfig/business-formation.json

```diff
@@ -242,6 +262,23 @@
         "radioYesText": "Yes",
         "error": "Select 'Yes' or 'No'."
       },
+      "isVeteranNonprofit": {
+        "label": "Non-Profit Veteran Corporation",
+        "description": "Is your Domestic Non-Profit a `Veteran Corporation|veteran-corporation`?",
+        "radioNoText": "No",
+        "radioYesText": "Yes",
+        "error": "Select 'Yes' or 'No'.",
+        "domNpVetCorpReviewText": "**Yes**, my Domestic Non-Profit is a Veteran Corporation.",
+        "yesBoardMembersReviewText": "**Yes**, my Non-Profit will have board members.",
```

Comment:
[boulder] Why are all these board member provision review text keys nested under `isVeteranNonprofit`? They are not related to the veteran question and should be in a key for the correct question.

Reply:
I see that the keys like `hasNonprofitBoardMembers` exists below - I assume the review text should go with its correct question, right? 

We can talk more about organizing this config properly if needed.